apps/editing-toolkit/editing-toolkit-plugin/wpcom-block-editor-nav-sidebar/src/components/toggle-sidebar-button/style.scss

```
@@ -1,9 +1,15 @@
 @import '~@wordpress/base-styles/variables';
+@import '~@wordpress/base-styles/mixins';
+@import '../nav-sidebar/style.scss';
 
 .wpcom-block-editor-nav-sidebar-toggle-sidebar-button__button {
 	height: $header-height !important;
 
+	// Match this animation with the nav-sidebar, so that the button is not hidden
+	// before the sidebar menu finishes sliding out.
 	&.is-hidden {
 		visibility: hidden;
+		animation: visibility $sidebar-transition-period linear;
+		@include reduce-motion( 'animation' );
```

From my tests, the icon button used to open the sidebar still gets hidden before the sidebar finishes animating.

Did you mean to use the CSS `transition` property, instead of `animation` ?


```suggestion
	// Match this animation with the nav-sidebar, so that the button is not hidden
	// before the sidebar menu finishes sliding out.
	&.is-hidden {
		visibility: hidden;
		transition: visibility $sidebar-transition-period linear;
		@include reduce-motion( 'transition' );
```

Using `transition` should delay the icon button's disappearance correctly (although the button will still disappear abruptly, since the `visibility` property can't be animated gradually)

---

Yes, it was a copy paste mistake. Now updated.